examples/amp-list.amp.html

```diff
@@ -57,6 +57,24 @@
     </template>
   </amp-list>
 
+  <!-- ## Using a script to enclose the template. -->
+  <amp-list width="auto" height="100" layout="fixed-height" src="comments.example.json">
+    <script type="amp-mustache">
```

Comment:
`type="text/plain" template="amp-mustache"`, right? Did we test that this example works?

Let's also add an integration test to `extensions/amp-list/0.1/test/integration/test-amp-list.js` to make sure it doesn't stop working. :)